v2/AdminCore.Services/EventService.cs

```diff
@@ -174,5 +174,39 @@ private Employee GetEmployeeFromEmployeeId(int employeeId)
       var employee = DatabaseContext.EmployeeRepository.GetSingle(x => x.EmployeeId == employeeId);
       return employee;
     }
+
+    public void IsHolidayValid(int employeeId, EventDateDto eventDates, bool modelIsHalfDay)
+    {
+      if (IsHolidayDatesAlreadyBooked(employeeId, eventDates))
+        throw new Exception("Already Booked");
```

Comment:
Not sure if we can to throw an exception? Just build up a list of messages and return to front end